docs/brand/copywriting/concepts.html

```diff
@@ -0,0 +1,54 @@
+---
+layout: page
+title: Concepts guidelines
+description: Quick guidelines for how to identify and refer to different conceptual units present on Stack Overflow
+---
+
+<section class="stacks-section">
+    {% header "h2", "Post scores" %}
+    <ul class="stacks-list">
+        <li>Questions, answers, and other types of Posts have <em>scores</em>.</li>
+        <li>A post’s <em>score</em> is calculated by subtracting the total number of downvotes from the total number of upvotes.</li>
+        <li>In almost all cases when referring to the sum of votes, we will use the term <em>score</em>.</li>
+        <li>
+            Use of the  term <em>votes</em> to refer to the post’s score is only acceptable in cases where we need to present the data in the format 
+            <code>{ScoreValue} votes</code> (as is done in the <a href="/product/components/post-summary/">Post summary</a>). In this case, "votes" 
```

Comment:
Just being nit-picky!

```suggestion
            <code>{ScoreValue} votes</code> (as is done in the <a href="/product/components/post-summary/">post summary component</a>). In this case, "votes" 
```